lib/toolkit/loader.js

```diff
@@ -785,7 +786,20 @@ function Loader(options) {
     if (isNative && !uri)
       uri = id;
     let module = Module(id, uri);
-    module.exports = freeze(modules[id]);
+
+    // Lazily expose built-in modules in order to
+    // allow them to be loaded lazily.
+    let cachedModules = modules;
```

Comment:
Oops yes that was misleading... I put the const, hopefully this is clear enough now?